### DISCUSSION TITLE
DragOverlay and shadowed preview

### DISCUSSION THREAD

**Discussion by bloodbird** [clauderic/dnd-kit#446] [Q&A]
Hello and thanks for all the work on the dnd-kit!
I've migrated from some old versions of @dnd-kit/core+sortable to the latest @dnd-kit/sortable (that has @dnd-kit/core deps) and it fixed some bugs I had when autoScrolling with dnd-kit.

What I can't manage to bring back now, is the DragOverlay shadowed preview in a vertical sorted list when Dragging without releasing. It used to display the future location as a shadow, moving elements around the future position, out of the box, but now it just doesn't.

Could a gentle soul point me to the right parameter I am missing or the behavior name of what I am looking for?

Ps: I've initially posted this in Issues but then noticed the Discussions tab (my bad, I closed the issue)

**Comment by clauderic** [comment]
> What I can't manage to bring back now, is the DragOverlay shadowed preview in a vertical sorted list when Dragging without releasing. It used to display the future location as a shadow, moving elements around the future position, out of the box, but now it just doesn't.

This was never a feature of `<DragOverlay>`, it's a feature of `useSortable` when you use `<DragOverlay>`, the active sortable item will move to its new position. It's up to you to style it with a lower opacity.

Nothing about this behaviour has changed recently as far as I can remember. 

Could you share a CodeSandbox replicating the issue? It's hard to provide more specific guidance without seeing the code.

**Reply by bloodbird** [reply]
Thank you Clauderic for the reply and for being so active on this project!

So I understand this is related mainly to useSortable and what I don't understand is that I didn't touch any code, but just upgraded the packages. So I would think it's not just about styling or opacity, as the sorting behavior (the moises effect or the element moving to its new position) doesn't happen at all.

I don't have enough time to create the code sandbox at the moment, so I may come back later. Meanwhile, I have found that the @dnd-kit/sortable : 2.0.1 working with @dnd-kit/core 2.1.2 has the fixes I needed for the autoScrolling and the sorting preview behavior works as intended, so I used these ones to move forward. I will probably try to recreate the bug in a CodeSandbox when I have a bit more time.
Thanks again!
